integration/feature/init/src/MillInitSbtTests.scala

```diff
@@ -0,0 +1,332 @@
+    - originally sbt 1.10.0
+     */
+    val url = "https://github.com/tototoshi/scala-csv/archive/refs/tags/2.0.0.zip"
+
+    test - integrationTest(url) { tester =>
+      bumpSbtTo1107(tester.workspacePath)
+
+      // Cross builds are not supported yet.
+      tester.testMillInit(
+        expectedCompileTasks = Some(SplitResolvedTasks(Seq(), Seq("compile", "test.compile"))),
+        expectedTestTasks = Some(SplitResolvedTasks(Seq(), Seq("test")))
+      )
+    }
+  }
+}
+
+object MillInitSbtScalaCsv136Tests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 28 KB
+    - originally sbt 1.2.8
+     */
+    val url = "https://github.com/tototoshi/scala-csv/archive/refs/tags/1.3.6.zip"
+
+    test - integrationTest(url) { tester =>
+      bumpSbtTo1107(tester.workspacePath)
+
+      tester.testMillInit(
+        expectedCompileTasks = Some(SplitResolvedTasks(
+          Seq("compile", "test.compile"),
+          Seq.empty
+        )),
+        expectedTestTasks = Some(SplitResolvedTasks(
+          Seq(),
+          /*
+          Paths relative to the workspace are used in the test sources such as `new File("src/test/resources/simple.csv")`
+          and they seem to cause the test to fail with Mill:
+          ```text
+          java.io.FileNotFoundException: src/test/resources/simple.csv (No such file or directory)
+          ```
+           */
+          Seq("test")
+        ))
+      )
+    }
+  }
+}
+
+// same as the one in the unit tests
+object MillInitSbtMultiProjectExampleTests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 12 KB
+    - originally sbt 1.0.2
+     */
+    val url =
+      "https://github.com/pbassiner/sbt-multi-project-example/archive/152b31df9837115b183576b0080628b43c505389.zip"
+
+    test - integrationTest(url) { tester =>
+      bumpSbtTo1107(tester.workspacePath)
+
+      val submodules = Seq("common", "multi1", "multi2")
+      if (System.getProperty("java.version").split('.').head.toInt <= 11)
+        tester.testMillInit(
+          expectedCompileTasks = Some(SplitResolvedTasks(
+            Seq("compile") ++ submodules.flatMap(_.allCompileTasks),
+            Seq.empty
+          )),
+          expectedTestTasks = Some(SplitResolvedTasks(submodules.map(_.testTask), Seq.empty))
+        )
+      else
+        tester.testMillInit(
+          // initCommand = defaultInitCommand ++ Seq("--jvm-id", "11"),
+          expectedCompileTasks = Some({
+            /*
+            `multi1.compile` doesn't work well when Mill is run with JDK 17 and 21:
+            ```text
+            1 tasks failed
+            multi1.compile java.io.IOError: java.lang.RuntimeException: /packages cannot be represented as URI
+                java.base/jdk.internal.jrtfs.JrtPath.toUri(JrtPath.java:175)
+                scala.tools.nsc.classpath.JrtClassPath.asURLs(DirectoryClassPath.scala:183)
+                ...
+            ```
+            Passing a `jvmId` 11 doesn't work here.
```

Comment:
Does this work with JDK 11? If so, could we try writing a `.mill-jvm-version` file to `temurin:11` to see if that makes it work?

Reply:
Yes it does. I will give it a try.